src/pyosmeta/parse_issues.py

```diff
@@ -221,6 +221,25 @@ def _postprocess_meta(self, meta: dict, body: List[str]) -> dict:
 
         return meta
 
+    def _postprocess_labels(self, meta: dict) -> dict:
+        """
+        Handle specific labels that are model properties
```

Comment:
please be more specific with the docstrings. Essentially this package can be hard to "dig back into" so let's be specific about what this helper does since we don't have great docs for the package yet. 

Let's be specific about it's purpose in the docstring

Reply:
I clarified the docstrings in b9868eb